azure-pipelines.yml

```diff
@@ -0,0 +1,25 @@
+# see https://docs.microsoft.com/en-us/azure/devops/pipelines/process/resources?view=azure-devops&tabs=schema#define-a-repositories-resource
+resources:
+  repositories:
+    - repository: IOMDevopsRepo
+      type: git
+      name: order-iom-partner-devops
+      ref: task/76533-ci-pipeline
+
+trigger:
+  branches:
+    include:
+      - '*'
```

Comment:
as I understand, triggered by any change in any branch

Reply:
Yes. But this part of the pipeline is fully in the hands of partners (you are the partner of the Blueprint project). The partner can simply change this setting.